### ISSUE TITLE
KS Hairdos SSE

### ISSUE THREAD

**Issue by SynopsiHub** [tiltedphoques/Mod-Compatibility#34]
# Mod for adding many hairstyle options to the character creator.

#### Dependencies

None

# Compatibility

> Mod is compatible completely, 0 major bugs.

# Additional Information

> If there any issues regarding your character hair changing, make sure you are using "showracemenu" once getting past Helgen or finish alternate start, save your character, THEN be the first to join the connecting server. Others must do the same. If you try to make said changes while on a live server, it will not reflect correctly. Edit: This is intentional. ST does not update racemenu changes while live.

**Comment by Dragonissa** [comment]
We dont sync any racemenu changes, hence why doing it online wont work.

**Comment by SynopsiHub** [comment]
> We dont sync any racemenu changes, hence why doing it online wont work.

Understood! I'll edit the post.

**Comment by lore-forged** [comment]
My wife and I are having an odd issue with this: when we load in, our hair colors match. On my screen, she has my hair color, and on her screen I have hers. Has anyone else had this issue?

Edit: Custom hair colors caused this problem. That's all!

**Comment by mavericks52** [comment]
Important thing to not the smp version makes everyone bald

**Comment by PseudoHyena** [comment]
> Important thing to not the smp version makes everyone bald

It works just fine with smp. I had the same issue and the reason was that the mod was marked as Light (see Plugins menu in Vortex). Press Mark as Regular and it's going to work. The same goes for all mods, they shouldn't be marked as Light.

**Comment by chalamander** [comment]
Yep. You can get KSHairdos SMP working in Mod Organiser 2 by opening the mod in SSEEdit and deleting the 'ESL' line.

**Comment by chalamander** [comment]
Actually after further testing, using the SMP version of KS Hairdos is causing crashes upon entering new areas in-game. 
Seems like it's not compatible :(

**Comment by PseudoHyena** [comment]
> Actually after further testing, using the SMP version of KS Hairdos is causing crashes upon entering new areas in-game. Seems like it's not compatible :(

Yeah. same for me. It happens when to players with smp hairs are nearby. I looked at call stack and I found that it's indeed crash on physics side.
